File: 백준/Bronze/2752. 세수정렬/세수정렬.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String[] str = sc.nextLine().split(" ");
        int a = Integer.parseInt(str[0]);
        int b = Integer.parseInt(str[1]);
        int c = Integer.parseInt(str[2]);

        int[] arr = {a, b, c};

        Arrays.sort(arr);

        System.out.println(arr[0] + " " + arr[1] + " " + arr[2]);
    }
}

File: 백준/Bronze/2752. 세수정렬/세수정렬.cs
string[] str = Console.ReadLine().Split();

int a = int.Parse(str[0]);
int b = int.Parse(str[1]);
int c = int.Parse(str[2]);

int[] arr = {a, b, c };

Array.Sort(arr);

Console.WriteLine($"{arr[0]} {arr[1]} {arr[2]}");